proto/kava/pricefeed/v1beta1/pricefeed.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+package kava.pricefeed.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/kava-labs/kava/x/pricefeed/types";
+
+// Market defines an asset in the pricefeed.
+message Market {
+  option (gogoproto.goproto_stringer) = false;
+
+  string          market_id   = 1 [(gogoproto.customname) = "MarketID"];
+  string          base_asset  = 2;
+  string          quote_asset = 3;
+  repeated string oracles     = 4;
+  bool            active      = 5;
+}
+
+// PostedPrice defines a price for market posted by a specific oracle.
+message PostedPrice {
+  option (gogoproto.goproto_stringer) = false;
+
+  string market_id      = 1 [(gogoproto.customname) = "MarketID"];
```

Comment:
should we consider updating our code to use `Id` instead? We have a number of these vars and it'll be good to just generate them without overriding with `customname`
cc @rhuairahrighairidh @denalimarsh

Reply:
@nddeluca mentioned using it since it follows I believe Go initialism styling (?) and reduces the diff.  Although it sometimes leads to inconsistency since it can't be used for fields that are referenced in grpc-gateway query paths. So still not 100% sure on keeping it